shards/shards.go

```diff
@@ -971,6 +971,14 @@ func mkRankedShard(s zoekt.Searcher) *rankedShard {
 			if priority > maxPriority {
 				maxPriority = priority
 			}
+
+			// No one is reading s yet, so we can mutate it.
```

Comment:
`s`?

Reply:
s is the name of the passed in shard. Will make this clearer, my bad.

Reply:
Ah I just didn't see it in the diff 😮‍💨

Reply:
sorry I said I would change it then I just went ahead and merged. Will follow up.